config/olm-catalog/ibm-spectrum-scale-csi-operator/2.5.0/ibm-spectrum-scale-csi-operator.v2.5.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,484 @@
+        x-descriptors:
+        - urn:alm:descriptor:io.kubernetes:Secret
+#      - description: An internal counter used by the operator to trigger reloads on
+#          secret change.
+#        displayName: Secret Counter
+#        path: secretCounter
+#        x-descriptors:
+#        - urn:alm:descriptor:com.tectonic.ui:hidden
+#      - description: A generic trigger to be used in kickin off the ansible operator.
+#        displayName: trigger
+#        path: trigger
+#        x-descriptors:
+#        - urn:alm:descriptor:com.tectonic.ui:hidden
+      - description: Mapping of K8s node with IBM Spectrum Scale node.
+        displayName: nodeMapping
+        path: nodeMapping
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+      - description: tolerations to distribue to IBM Spectrum Scale CSI pods.
+        displayName: tolerations
+        path: tolerations
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+      - description: Node selector for IBM Spectrum Scale CSI Plugin pods.
+        displayName: Plugin Node Selector
+        path: pluginNodeSelector
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+      - description: Key for node selector.
+        displayName: Key
+        path: pluginNodeSelector.key
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:label
+      - description: Value for key.
+        displayName: Value
+        path: pluginNodeSelector.value
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:label
+      - description: Node selector for provisioner sidecar pods.
+        displayName: Provisioner Node Selector
+        path: provisionerNodeSelector
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+      - description: Key for node selector.
+        displayName: Key
+        path: provisionerNodeSelector.key
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:label
+      - description: Value for key.
+        displayName: Value
+        path: provisionerNodeSelector.value
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:label
+      - description: Node selector for attacher sidecar pods.
+        displayName: Attacher Node Selector
+        path: attacherNodeSelector
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+      - description: Key for attacher node selector pods.
+        displayName: Key
+        path: attacherNodeSelector.key
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:label
+      - description: Value for attacher node selector key.
+        displayName: Value
+        path: attacherNodeSelector.value
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:label
+      - description: Node selector for snapshotter sidecar pods.
+        displayName: Snapshotter Node Selector
+        path: clusters
+        path: snapshotterNodeSelector
```

Comment:
mutiple `path` mentioned.

Reply:
This was a copy of 2.4.0 CSV which had multiple paths.